frontend/src/metabase/visualizations/components/ChoroplethMap.jsx

```diff
@@ -254,10 +255,7 @@ export default class ChoroplethMap extends Component {
             column: cols[metricIndex],
             dimensions: [
               {
-                value:
-                  feature != null
-                    ? getFeatureName(feature)
-                    : row[dimensionIndex],
+                value: row[dimensionIndex],
```

Comment:
The old code was wrong. We shouldn't use "feature name" in dimensions since data will be used for the tooltips.